app/serializers/ecf_user_serializer.rb

```diff
@@ -63,7 +63,7 @@ class ECFUserSerializer
   end
 
   attributes :cohort do |user|
-    user.cohort.start_year
+    user.cohort&.start_year
```

Comment:
```suggestion
    find_school_cohort(user)&.cohort&.start_year
```

Reply:
🤦 Can't believe I missed the method right there